### PR TITLE
SWEN-43: Write test case specification for SWEN-20

### PR DESCRIPTION
**Pull request by carolc-p**
The Get Vaccinated app shall notify ineligible users, who attempt to schedule an appointment, to check back later.

### CODE REVIEW

File: src/test/java/com/sweng894/GetVaccinated/PersonTests.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		person = new Person("John Doe", "2", "N");
+	}
+
+	@Test
+	void testEligiblePerson() {
+
+		// Person has not been vaccinated and is not in the current rollout phase
+		if (("1B".compareTo(person.getPhase())) <= 0) {
+			assertTrue("N".equals(person.getIsVaccinated()), person.getName() + " is ineligible.");
+		}
```
Comment (rsolomo) [inline src/test/java/com/sweng894/GetVaccinated/PersonTests.java:26]: Having conditions in the test code can make it more difficult to tell what is being tested. We probably should have the logic inside the class under test, and have test cases that ensure the branches are covered.

**Review by rsolomo — APPROVED** [review]
Test coverage is good. A complete implementation of SWEN-20 would involve having more than a class of getters and setters, but this is fine as a start.

**Comment by carolc-p** [comment]
Will add more tests as more code gets implemented.

File: src/test/java/com/sweng894/GetVaccinated/PersonTests.java (hunk 1 above)
Comment (carolc-p) [reply src/test/java/com/sweng894/GetVaccinated/PersonTests.java:26]: What do you mean by having the log "inside the class under test"? Have the logic in the Person.java file?

Comment (rsolomo) [reply src/test/java/com/sweng894/GetVaccinated/PersonTests.java:26]: Yes, the if statement would be somewhere in the Person.java file.

**Comment by carolc-p** [comment]
I will merge and close this for now. I will put additional test cases that come about with the source code pull request.

**Pull request merged**